Sources/main.swift

```diff
@@ -12,7 +12,7 @@ import SwiftGenKit
 
 // MARK: Common
 
-let templatesRelativePath = (Bundle.main.resourcePath ?? ".") + "/templates"
+let templatesRelativePath = "../templates"
```

Comment:
Maybe at some point we should extract that constant into its own file, then make the Rakefile reset just that file at the end of the build? 
I don't want to `git checkout main.swift` because if we made local changes to main and forgot to stage and commit them before testing our changes with `rake install` locally we'd lose all our changes. But if there constant is isolated in a one-line file it should be ok.